portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,11 +37,12 @@
 /** Servlet that returns data for my portfolio*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
-  private Comments comments = new Comments(5, new ArrayList<String>());
-
+  private static final int COMMENT_LIMIT_DEFAULT = 5;
+  private static final String COMMENT_LANGUAGE_CODE_DEFAULT = "en";
+  private Comments comments = new Comments(COMMENT_LIMIT_DEFAULT,
+                                           new ArrayList<String>(),
+                                           COMMENT_LANGUAGE_CODE_DEFAULT);
   private Gson gson = new Gson();
```

Comment:
There is no need for this member variable. You can initialize it in where it's used below.